lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,73 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+
+                if ($('#accordion').length) {
+
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                $('#open_close_accordion a').click();
+                $('.course-wrapper').removeClass('closed');
+                $('#accordion').show();
+
+                navigation.checkForCurrent();
+                navigation.listenForClick();
+            },
+
+            checkForCurrent: function() {
+                var active = $('#accordion div div:has(a.active)').index('#accordion div div') ? $('#accordion div div:has(a.active)').index('#accordion div div') : 0;
```

Comment:
These selectors are long and bound tightly to DOM layout, can you please update to add specific classes to the DOM and use these to select elements?